views/scan.ejs

```diff
@@ -109,6 +109,31 @@
       </datalist>
       <table class="table table-bordered">
         <tbody>
+          <tr>
+            <th>
+              <select name="operationType">
+                <option value="scan"<% if (query.operationType === 'scan') { %> selected<% } %>>Scan</option>
+                <option value="query"<% if (query.operationType === 'query') { %> selected<% } %>>Query</option>
+              </select>
+            </th>
+            <td>
+              <select name="queryableSelection">
+                <option value="table"<% if (query.queryableSelection === 'table') { %> selected<% } %>>[Table] <%= Table.TableName %></option>
```

Comment:
I feel that it's a bit redundant to show table name if there are no `GlobalSecondaryIndexes` since table name is shown already at the top of the page.

![Screenshot 2020-02-05 at 00 12 44](https://user-images.githubusercontent.com/153197/73796153-cc8d9800-47ac-11ea-94a4-2c9e3de7e35d.png)

(that dropdown doesn't have any other items)

Reply:
I am not sure if hiding it in this case makes a lot of sense though, I based some of this setup on the DynamoDB Console built into AWS Management Console.

If you want me to hide it I can do that change, for now, I've just disabled the select when there are no indexes available.